gbfs.md

```diff
@@ -311,9 +312,9 @@ The following fields are all attributes within the main `data` object for this f
 
 Field Name | REQUIRED | Type | Defines
 ---|---|---|---
-`datasets` | Yes | Array | An array of objects, each containing the keys below.
+`datasets` | Yes | Array<Object> | 
```

Comment:
I prefer when all fields have a definition, even though it can be redundant for the fields which are containers for objects (this is a personal preference).

Curious to know what @josee-sabourin and others think.

```suggestion
`datasets` | Yes | Array<Object> | An array of `datasets` produced by a publisher.
```

Reply:
I've push a commit in that direction: https://github.com/MobilityData/gbfs/pull/576/commits/5d6ad389a1be917eb4215e75a5847bb77f52f74a